app/routes/index.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  // beforeModel(){
```

Comment:
Don't have commented out code inside pull requests. Delete it - (even if the next issue calls for it)